demisto_sdk/commands/generate_outputs/generate_context/tests/generate_integration_context_test.py

```diff
@@ -136,10 +129,10 @@ def test_generate_integration_context(mocker, tmpdir):
         if command.get('name') == command_name:
             command['outputs'] = ''
             break
+    else:
+        assert False, 'command is not found in yml_data'
 
-    generate_integration_context.generate_integration_context(filename,
-                                                              FAKE_EXAMPLES_FILE,
-                                                              insecure=True,
+    generate_integration_context.generate_integration_context(filename, FAKE_EXAMPLES_FILE, insecure=True,
```

Comment:
do you really need to pass insecure in a test?

Reply:
not really, deleted